addons/website/views/website_templates.xml

```diff
@@ -395,16 +394,15 @@
         <li class="list-inline-item">
             <div class="dropup">
                 <button class="btn btn-sm btn-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                    <span t-esc="list(filter(lambda lg : lg[0] == lang, languages))[0][1].split('/').pop()"/>
+                    <span t-esc="list(filter(lambda lg : lg[0] == lang, languages))[0][2].split('/').pop()"/>
                     <span class="caret ml4"/>
                 </button>
                 <div class="dropdown-menu" role="menu">
                     <t t-foreach="languages" t-as="lg">
-                        <a t-att-href="url_for(request.httprequest.path + '?' + keep_query(), lang=lg[0])"
+                        <a t-att-href="url_for(request.httprequest.path + '?' + keep_query(), lg[0])"
```

Comment:
keep lang=

Reply:
Alright, added it back. Named `lang_code` now.
Tried to name variable throughout this PR so it is easy to figure what the variable is holding or what the parameter is expecting: code or url_code.
Otherwise, everything was named lang and you had no clue if it was a lang record, a lang.code or a lang.url_code